sgm/data/dataset.py

```diff
@@ -1,20 +1,22 @@
+import logging
 from typing import Optional
 
 import torchdata.datapipes.iter
 import webdataset as wds
 from omegaconf import DictConfig
 from pytorch_lightning import LightningDataModule
 
+log = logging.getLogger(__name__)
```

Comment:
@akx can you rename log --> logger everywhere?

Reply:
Done.